lib/img/local.js

```diff
@@ -43,6 +43,22 @@ LocalDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function localDelete(key) {
+    return new P(function deletePromise(resolve, reject) {
```

Comment:
I believe I was hitting an error, and wanted to improve the stack trace. My preference is to name as many functions as I can, because it doesn't hurt and only ever improves stack traces...